components/crud-web-apps/jupyter/backend/requirements.txt

```diff
@@ -1 +1,2 @@
+kubernetes==v22.6.0
```

Comment:
Could you instead make this change into the common code, so that it takes effect for all the web apps?

https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/common/backend/setup.py#L6

Reply:
> @benjamintanweihao could you verify that deleting objects [i.e. Notebooks] also works as expected?
> 
> I remember that there was a change in the newer version of the library on how to pass the deletion-policy, but I might be wrong.

Just tried deleting the notebook and it's working :D. Sure thing, I'll also make the change everywhere.

Reply:
Done. :)